.travis.yml

```diff
@@ -1,9 +1,20 @@
 sudo: false
 language: python
+
 python:
   - 3.5
+
 services:
-    - postgresql
+  - postgresql
+  - docker
+
+env:
+  global:
+    - secure: "" # DOCKER_EMAIL
```

Comment:
I just discovered this feature too, travis has something to encrypt sensible data! You learn more [here](https://docs.travis-ci.com/user/encryption-keys/).